base/abstractarraymath.jl

```diff
@@ -137,13 +137,13 @@ julia> b = [1 2; 3 4]
  1  2
  3  4
 
-julia> flipdim(b,2)
+julia> reverse(b, dims=2)
 2×2 Array{Int64,2}:
  2  1
  4  3
 ```
 """
-function flipdim(A::AbstractArray, d::Integer)
+function reverse(A::AbstractArray; dims::Integer)
     nd = ndims(A)
     1 ≤ d ≤ nd || throw(ArgumentError("dimension $d is not 1 ≤ $d ≤ $nd"))
```

Comment:
This should be updated to reflect the change from the argument being called `d` to being called `dims`.

Reply:
ooops